tests/migration_test.go

```diff
@@ -1155,7 +1155,7 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 				table.Entry("[test_id:2226]with ContainerDisk", newVirtualMachineInstanceWithFedoraContainerDisk),
 				table.Entry("[test_id:2731]with OCS Disk", newVirtualMachineInstanceWithFedoraOCSDisk),
 			)
-			It("[test_id:3241]should be able to cancel a migration right after posting it", func() {
+			FIt("[test_id:3241]should be able to cancel a migration right after posting it", func() {
```

Comment:
`FIt` does not work if  a `--skip` or `--focus` is added to the test execution and that is what we do in `automation/test.sh`.